extension/js/common/platform/xss.ts

```diff
@@ -151,7 +151,9 @@ export class Xss {
             a.setAttribute('data-test', 'show-inline-image');
             Xss.replaceElementDANGEROUSLY(img, a.outerHTML); // xss-safe-value - "a" was build using dom node api
           } else {
-            Xss.replaceElementDANGEROUSLY(img, `<span>[Remote images are blocked due to security]</span>`); // xss-safe-value
+            img.setAttribute('data-src', img.getAttribute('src') ?? '');
+            img.classList.add('replace_to_base64_image');
+            img.setAttribute('src', '/img/loading.gif');
```

Comment:
let's use our standard loader `UI.spinner` - it's more lightweight as uses `svg`

Reply:
Ok, Sounds good